src/controllers/class.controller.js

```diff
@@ -35,7 +35,7 @@ export const createClass = (req, res) => {
       if (err) {
         debugLog(err);
         if (err.code === "ER_DUP_ENTRY") {
-          return res.send({
+          return res.status(409).send({
```

Comment:
Check the `logic written in FE` and verify by doing this it still works!!
Otherwise please undo this change.